### DISCUSSION TITLE
FreshRSS Android App (1.4.1-2)

### DISCUSSION THREAD

**Discussion by GenerallyDisjointed** [FreshRSS/FreshRSS#5997] [Q&A] [answered]
**Describe the bug**
I selfhost FreshRSS (1.22.1) on a vps, and both my api endpoint tests pass.

On my android phone, I downloaded the FreshRSS app and entered my full URL, username and api password. When it connects I can see all the rss feeds loading with a count beside each one, however, when I tap through to an individual feed, I see these:

There are no articles from [Feedlname] yet. New articles will probably be published soon.
-OR-
You've read every article from [Feedname]

**To Reproduce**
1) Download and use FreshRSS 1.4.1-2
2) Login with url https://domain and username/password
3) Let it populate with feeds
4) Tap a feed to view items
5) See message

**Expected behavior**
Tapping a feed should display the feed items to view

**Screenshots**

**Environment information (please complete the following information):**
- Device: [Redmi 20 pro]
- OS: [LineageOS]
- Browser: [Mull 120.0.0]
- FreshRSS version: [1.22.1]
- Installation type: [e.g. Docker latest image]

**Additional context**
None

**Comment by Alkarex** [comment, marked as answer]
Hello,
For issues related to Christophe Henry's app, please refer to https://gitlab.com/christophehenry/freshrss-android
It is unrelated to the FreshRSS here.

Here, we can help you check that the API works correctly. Check in particular that documentation page https://freshrss.github.io/FreshRSS/en/developers/06_GoogleReader_API.html

You can also double-check with another app, to help finding out whether the bug is in the app or in FreshRSS:
https://github.com/FreshRSS/FreshRSS/blob/edge/README.md#apis--native-apps
